bam/src/monitoring_stream.cc

```diff
@@ -382,7 +363,7 @@ void monitoring_stream::_rebuild() {
   {
     std::ostringstream query;
     query << "SELECT ba_id"
-          << "  FROM " << (_db_v2 ? "mod_bam" : "cfg_bam")
+          << "  FROM mod_bam"
           << "  WHERE must_be_rebuild='1'";
```

Comment:
It would be better to directly initialize the string now.

Maybe in the two previous changes too, but I cannot see all the affectation.